third_party/k32w_sdk/k32w_sdk.gni

```diff
@@ -0,0 +1,209 @@
+      "${k32w_sdk_root}/middleware/wireless/framework/LED/Interface",
+      "${k32w_sdk_root}/middleware/wireless/framework/Lists",
+      "${k32w_sdk_root}/middleware/wireless/framework/MemManager/Interface",
+      "${k32w_sdk_root}/middleware/wireless/framework/OSAbstraction/Interface",
+      "${k32w_sdk_root}/middleware/wireless/framework/Panic/Interface",
+      "${k32w_sdk_root}/middleware/wireless/framework/PDM/Include",
+      "${k32w_sdk_root}/middleware/wireless/framework/RNG/Interface",
+      "${k32w_sdk_root}/middleware/wireless/framework/SerialManager/Source",
+      "${k32w_sdk_root}/middleware/wireless/framework/TimersManager/Interface",
+      "${k32w_sdk_root}/middleware/wireless/framework/TimersManager/Source",
+      "${k32w_sdk_root}/middleware/wireless/framework/XCVR/DK6",
+      "${k32w_sdk_root}/middleware/wireless/framework/XCVR/DK6/Build/Include",
+      "${k32w_sdk_root}/middleware/wireless/ieee-802.15.4/uMac/Include",
+      "${k32w_sdk_root}/rtos/amazon-freertos/lib/FreeRTOS/portable/GCC/ARM_CM3",
+      "${k32w_sdk_root}/rtos/amazon-freertos/lib/include",
+      "${k32w_sdk_root}/rtos/amazon-freertos/lib/include/private",
+      "${k32w_sdk_root}/rtos/amazon-freertos/lib/third_party/unity/src",
+    ]
+
+    libs = [
+      "${k32w_sdk_root}/middleware/wireless/ieee-802.15.4/lib/libMiniMac.a",
+      "${k32w_sdk_root}/middleware/wireless/framework/PDM/Library/libPDM.a",
+      "${k32w_sdk_root}/middleware/wireless/framework/XCVR/lib/libRadio.a",
+    ]
+
+    defines = [
+      "CPU_K32W061HN",
+      "CPU_JN518X",
+      "CPU_JN518X_REV=2",
+      "JENNIC_CHIP_FAMILY_NAME=_JN518x",
+      "gPWR_LDOMEM_0_9V_PD=0",
+      "SDK_DEBUGCONSOLE=0",
+      "NO_SYSCORECLK_UPD=0",
+      "USE_RTOS=1",
+      "FSL_RTOS_FREE_RTOS=1",
+      "MBEDTLS_FREESCALE_FREERTOS_CALLOC_ALT=1",
+      "gUartDebugConsole_d=0",
+      "gUsePdm_d=1",
+      "gLEDSupported_d",
+      "gKeyBoardSupported_d",
+      "gPdmNbSegments=63",
+      "osCustomStartup=1",
+      "OM15082=1",
+      "gKBD_KeysCount_c=3",
+      "gLEDsOnTargetBoardCnt_c=2",
+      "gLED_InvertedMode_d=1",
+      "ENABLE_RAM_VECTOR_TABLE=1",
+      "gTMR_Enabled_d=1",
+      "gTimestamp_Enabled_d=0",
+      "gMainThreadStackSize_c=2048",
+      "K32W_LOG_ENABLED=1",
+      "CHIP_ENABLE_OPENTHREAD=1",
+      "PoolsDetails_c=_block_size_ 512 _number_of_blocks_ 2 _pool_id_(0) _eol_ _block_size_ 768 _number_of_blocks_ 1 _pool_id_(0) _eol_",
+      "OPENTHREAD_CONFIG_HEAP_EXTERNAL_ENABLE=1",
+      "SUPPORT_FOR_15_4=1",
+    ]
+
+    if (defined(invoker.defines)) {
+      defines += invoker.defines
+    }
+
+    cflags = [
+      "-Wno-unused-function",
+      "-Wno-conversion",
+    ]
+
+    # Now add our "system-header" include dirs
+    foreach(include_dir, _sdk_include_dirs) {
+      cflags += [ "-isystem" + rebase_path(include_dir, root_build_dir) ]
+    }
+  }
+
+  # TODO - Break up this monolith and make it configurable.
```

Comment:
should we create a GH issue for this and reference it here instead of a simple TODO?

Reply:
Created https://github.com/project-chip/connectedhomeip/issues/3219. How do I create a reference to it?